### ISSUE TITLE
Test fails in Success.js

### ISSUE THREAD

**Issue by chillyistkult** [ant-design/ant-design-pro#1133]
`npm test` test fails with

`Cannot find module 'components/Result' from 'Success.js'`

It looks like `roadhog test` ignores the alias that is defined in .`webpackrc.js `

If I change the path in `Success.js` to `'../../components/Result'` it works as expected.

**Comment by chillyistkult** [comment]
I will just share my findings with you because the problem is really weird:

1. I created a `.babelrc` file and installed `npm install --save-dev babel-plugin-module-resolver`

```
{
  "plugins": [
    ["babel-plugin-module-resolver", {
      "alias": {
        "components": "./src/components",
      }
    }]
  ]
}
```

Usually this should have been enough already to make it work, but it doesn't. After some investigation I found out that when you do `cd node_modules/roadhog` and run `npm install` from inside the roadhog folder, it suddenly works. I don't know how this is possible, because the resolved dependencies are the same. Maybe you have an idea @sorrycc ?
